src/corefx/System.Globalization.Native/localeNumberData.cpp

```diff
@@ -31,9 +47,259 @@ enum LocaleNumberData : int32_t
 
 /*
 Function:
+NormalizePattern
+
+Returns a numeric string pattern in a format that we can match against the appropriate managed pattern.
+*/
+UnicodeString* NormalizePattern(const UnicodeString *srcPattern, UnicodeString *destPattern, bool isNegative)
```

Comment:
Do you need to return the UnicodeString?  The caller of this method doesn't capture the return value.